movie-app/src/containers/movie-details-container/MovieDetailsContainer.tsx

```diff
@@ -1,35 +1,68 @@
-import React from "react";
-import MovieDetailsWrapper from "./MovieDetailsWrapper";
-import poster from "../../assets/images/netflix-streaming-vs-traditional-cable.jpg";
-import { IMovie } from "../../interfaces/IMovie";
+import React, { useEffect } from 'react';
+import MovieDetailsWrapper from './MovieDetailsWrapper';
+import { connect } from 'react-redux';
+import {
+  getMovieById,
+  loadMovies,
+  setSearchParams,
+} from '../../redux/rootActions';
+import { useLocation, useParams } from 'react-router';
+import EmptyState from '../../shared-components/empty-state/EmptyState';
 
-const MovieDetailsContainer = (props: IMovie) => {
-  const genres = (genres: string[]) => <span>{genres.join(",")}</span>;
+const MovieDetailsContainer = (props: any) => {
+  const location = useLocation();
+  const params: any = useParams();
 
-  return (
-    <MovieDetailsWrapper score={props.vote_average}>
+  useEffect(() => {
+    if (props.history.location.pathname.includes('film')) {
```

Comment:
same, why not use location from hook instead?

Reply:
done